packages/salesforce/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # @openfn/language-salesforce
 
+## 5.0.0-alpha
```

Comment:
Maybe we can add a single sentence here, between the version and `Major Changes`, like this:

> This is an experimental alpha release based on jsforce  beta 2. It is not recommended for production use.